packages/angular/package.json

```diff
@@ -47,11 +50,20 @@
     "@angular/cli": "~10.0.0-next.4",
     "@angular/compiler-cli": "~10.0.0-next.6",
     "@angular/language-service": "~10.0.0-next.6",
+    "@babel/core": "^7.9.6",
+    "@storybook/addon-actions": "^5.3.18",
```

Comment:
fyi - this needs to be upgrade when I tried the last time fresh I forgot to use the v6 storybook generator. That will be the next push but v6 is giving lots of issues to work through.